src/WasmDis.spec.ts

```diff
@@ -290,3 +290,107 @@ describe("NameSectionReader", () => {
     expect(nr.getVariableName(0, 1, false)).toBe("$y (;1;)");
   });
 });
+
+describe("WasmDisassembler.getResult() with function code", () => {
+  const watString =
+  `(module
+  (export "export.function" (func $f))
+  (func $f (result i32) i32.const 0)
+  (func $f1 (result i32) i32.const 1))`;
+  const fileName = `test.wat`;
+  const expectedLines = [
+    "(module",
+    "  (type $type0 (func (result i32)))",
+    "  (export \"export.function\" (func $func0))",
+    "  (func $func0 (result i32)",
+    "    i32.const 0",
+    "  )",
+    "  (func $func1 (result i32)",
+    "    i32.const 1",
+    "  )",
+    ")",
+  ];
+
+  test("addOffsets is true", () => {
+    const { buffer } = parseWat(fileName, watString).toBinary({
+      write_debug_names: true
+    });
+    const parser = new BinaryReader();
+    parser.setData(buffer.buffer, 0, buffer.byteLength);
+    const dis = new WasmDisassembler();
+    dis.addOffsets = true;
+    const offsetInModule = 0;
+    dis.disassembleChunk(parser, offsetInModule);
+    const result = dis.getResult();
+    expect(result.done).toBe(true);
+    expect(result.lines).toEqual(expectedLines);
+    expect(result.offsets).toEqual([0, 10, 22, 43, 46, 48, 49, 51, 53, 78,]);
+    expect(result.bodyFunctionOffsets).toEqual([[43, 49,], [49, 54,], ]);
```

Comment:
So then byte 49 is breakable, which is equivalent to the (func $func1 ... line? I think what we want here is the range of instructions.

Reply:
43, 49 and 54 are all unbreakable. As far as I can tell, function header is now one line only. So 
the range are exclusion.

Reply:
When there are local variables that are not parameters, they are declared on subsequent lines, but shouldn't be breakable.